src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -3421,6 +3421,170 @@ adjset_uniform(Node &res)
     }
 }
 
+//-----------------------------------------------------------------------------
+void 
+polychain(const index_t length, // how long the chain ought to be
+          Node &res)
```

Comment:
It looks like this might not be a fresh branch? (It also has the polychain code)?

I suggest starting a new branch fresh from develop, and replaying the commits related to mapping.